src/content/en/fundamentals/getting-started/primers/async-functions.md

```diff
@@ -0,0 +1,317 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Async functions allow you to write promise-based code as if it were synchronous
+
+{# wf_published_on: 2016-10-07 #}
+{# wf_updated_on: 2016-10-07 #}
+
+# Async functions - making promises friendly {: .page-title }
+
+{% include "web/_shared/contributors/jakearchibald.html" %}
+
+Async functions are enabled by default in Chrome 55 and they're quite frankly
+marvelous. They allow you to write promise-based code as if it were synchronous,
+but without blocking the main thread. They make your asynchronous code less
+"clever" and more readable.
+
+Async functions look like this:
+
+    async function myFirstAsyncFunction() {
+      try {
+        const fulfilledValue = await promise;
+      }
+      catch (rejectedValue) {
+        // …
+      }
+    }
+
+If you use the `async` keyword before a function definition, you can then use
```

Comment:
There are a few reasons:
- When you have such drastically different modes, it's a benefit to readability to have it called out when the function starts (as Jake says above).
- You can have an async function which doesn't contain any await; the semantics will be kinda sorta like if `return foo` was return `Promise.resolve(foo)`
- `await` is a new keyword, but we can't "just" add keywords. They might already be used as variables. Scoping the keyword-ness to async functions is one technique that `await` uses. (`async` uses other techniques, e.g., that if it's followed by `function`, it would've been a syntax error.)